packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -95,13 +95,9 @@ local function request(url, accept_404_reply, auth_token)
     return res, nil
 end
 
-local function is_ip_per_task(app)
-    return app["ipAddress"] ~= nil and app["ipAddress"] ~= cjson_safe.null
-end
-
-local function is_user_network(app)
+local function is_container_network(app)
     local container = app["container"]
-    return container and container["type"] == "DOCKER" and container["docker"]["network"] == "USER"
+    return container and app["networks"][1]["mode"] == "container"
```

Comment:
`networks` key may be not present[1], the specification does not mention that there is going to be at least one network entry in the `app["networks"][1]` list[1][2]


[1] https://github.com/mesosphere/marathon/blob/5d2b7970c147e11c8181971f381b179e773d3671/docs/docs/rest-api/public/api/v2/types/app.raml#L239-L241
[2] https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md#array-type